src/test/resources/application-test.yml

```diff
@@ -7,3 +7,9 @@ management.endpoint:
 
 hmpps-auth:
   url: "http://localhost:8090/auth"
+
+spring:
+  datasource:
+    url: 'jdbc:h2:mem:iis-db;MODE=MSSQLServer'
```

Comment:
would have preferred a local sqlserver instance running but couldn't find one for macs